test/unit/rules/inline-link-to-test.js

```diff
@@ -16,22 +16,38 @@ generateRuleTests({
   bad: [
     {
       template: "{{link-to 'Link text' 'routeName'}}",
+      fixedTemplate: "{{#link-to 'routeName' }}Link text{{/link-to}}",
 
       result: {
         message,
         source: "{{link-to 'Link text' 'routeName'}}",
         line: 1,
         column: 0,
+        isFixable: true,
       },
     },
     {
       template: "{{link-to 'Link text' 'routeName' one two}}",
+      fixedTemplate: "{{#link-to 'routeName' one two }}Link text{{/link-to}}",
 
       result: {
         message,
         source: "{{link-to 'Link text' 'routeName' one two}}",
         line: 1,
         column: 0,
+        isFixable: true,
+      },
+    },
+    {
+      template: "{{link-to (concat 'Hello' @username) 'routeName' one two}}",
+      fixedTemplate: "{{#link-to 'routeName' one two }}{{concat 'Hello' @username }}{{/link-to}}",
```

Comment:
I think this specific example could _theoretically_ remove `concat` (since `concat` specifically is needed inside mustaches, but not in the content position).

Seems fine for a later cleanup though...

Reply:
yep, probably fine to do this separately. also I'm wondering if it makes sense to do this as a dedicated `no-unnecessary-concat-helper` rule instead.

Reply:
Ya, sounds good to me!